packages/qdrant-loader/src/qdrant_loader/webhooks/server.py

```
@@ -0,0 +1,135 @@
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail="Webhook body must be valid JSON.",
+        ) from exc
+
+
+@app.post("/webhooks/projects/{project_id}/{source_type}/{source}")
+async def webhook_project_route(
+    project_id: str,
+    source_type: str,
+    source: str,
+    request: Request,
+    force: bool = False,
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)